.travis.yml

```diff
@@ -5,21 +5,20 @@ matrix:
     # Linux Python 2
     - sudo: required
       language: python
+      python: "2.7"
```

Comment:
"2.7" -> 2.7?

Reply:
Well,
I copy/pasted this from travis-ci doc https://docs.travis-ci.com/user/languages/python/#Specifying-Python-versions
If it has no impact I'm happy to change it